app/views/posts/_overwatch_2_indicator.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="warning warning--orange bg-overwatch-2">
+  <div>
+    <strong>Overwatch 2</strong> Beta Compatible
+
+    <em class="tooltip tooltip--right">
+      (?)
+      <div class="tooltip__content">
+        This code is verified to be compatible with the Overwatch 2 Beta.
+        There is no guarantee that codes will function between different beta phases.
+        A code that works in the Overwatch 2 Beta will not work in Overwatch 1.
```

Comment:
Similar to the rewording of the Overwatch 2 index page, I'd suggest something like:
```suggestion
        This code is tagged as being importable in Overwatch 2.
        It is possible the code was only valid for a previous beta, or that functionality has broken due to updates.
        A code that works in the Overwatch 2 Beta will not work in Overwatch 1.
```